dkc/core/admin/quota.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib import admin
+from django.db import models
+from django_admin_display import admin_display
+import humanize
+
+from dkc.core.models import Quota
+
+
+@admin.register(Quota)
+class QuotaAdmin(admin.ModelAdmin):
```

Comment:
Note to QA, please take a look at the running admin page.

Reply:
I took a look at the admin page, but it is broken for me on loading the quotas page:

```
Environment:


Request Method: GET
Request URL: http://localhost:8000/admin/core/quota/

Django Version: 3.1.6
Python Version: 3.8.7
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.sites',
 'girder_style',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'django.contrib.postgres',
 'whitenoise.runserver_nostatic',
 'django.contrib.staticfiles',
 'corsheaders',
 'django_filters',
 'rest_framework',
 'rest_framework.authtoken',
 'oauth2_provider',
 'drf_yasg',
 'django_extensions',
 'debug_toolbar',
 'dkc.core.apps.CoreConfig',
 's3_file_field',
 'guardian']
Installed Middleware:
['debug_toolbar.middleware.DebugToolbarMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'corsheaders.middleware.CorsMiddleware',
 'whitenoise.middleware.WhiteNoiseMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/options.py", line 614, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/utils/decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/sites.py", line 233, in inner
    return view(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/utils/decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/options.py", line 1693, in changelist_view
    cl = self.get_changelist_instance(request)
  File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/options.py", line 735, in get_changelist_instance
    return ChangeList(
  File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/views/main.py", line 99, in __init__
    self.queryset = self.get_queryset(request)
  File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/views/main.py", line 450, in get_queryset
    ) = self.get_filters(request)
  File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/views/main.py", line 152, in get_filters
    spec = field_list_filter_class(
  File "/usr/local/lib/python3.8/site-packages/django/contrib/admin/filters.py", line 435, in __init__
    raise ImproperlyConfigured(

Exception Type: ImproperlyConfigured at /admin/core/quota/
Exception Value: The list filter 'EmptyFieldListFilter' cannot be used with field 'user' which doesn't allow empty strings and nulls.
```
Request Method: | GET
-- | --
http://localhost:8000/admin/core/quota/
3.1.6
ImproperlyConfigured
The list filter 'EmptyFieldListFilter' cannot be used with field 'user' which doesn't allow empty strings and nulls.
/usr/local/lib/python3.8/site-packages/django/contrib/admin/filters.py, line 435, in __init__
/usr/local/bin/python
3.8.7
['/opt/django-project',  '/opt/django-project',  '/usr/local/lib/python38.zip',  '/usr/local/lib/python3.8',  '/usr/local/lib/python3.8/lib-dynload',  '/usr/local/lib/python3.8/site-packages',  '/opt/django-project']


Request Method: | GET
-- | --
http://localhost:8000/admin/core/quota/
3.1.6
ImproperlyConfigured
The list filter 'EmptyFieldListFilter' cannot be used with field 'user' which doesn't allow empty strings and nulls.
/usr/local/lib/python3.8/site-packages/django/contrib/admin/filters.py, line 435, in __init__
/usr/local/bin/python
3.8.7
['/opt/django-project',  '/opt/django-project',  '/usr/local/lib/python38.zip',  '/usr/local/lib/python3.8',  '/usr/local/lib/python3.8/lib-dynload',  '/usr/local/lib/python3.8/site-packages',  '/opt/django-project']

Reply:
Fixed.